cli/manage_test.go

```diff
@@ -0,0 +1,34 @@
+package cli
+
+import (
+	"flag"
+	mcli "github.com/codegangsta/cli"
```

Comment:
can you please not alias this to `mcli` so it's consistant with the rest of the codebase ? Thanks